pkg/drivers/vfkit/driver_darwin.go

```diff
@@ -467,9 +468,24 @@ func (d *Driver) sendSignal(s syscall.Signal) error {
 }
 
 func (d *Driver) supportsVirtiofs() bool {
-	macosVersion, _, err := crcos.RunWithDefaultLocale("sw_vers", "-productVersion")
+	supportsVirtioFS, err := macosNewerThan(11.0)
 	if err != nil {
-		return false
+		log.Debugf("Not able to compare version: %v", err)
 	}
-	return strings.HasPrefix(macosVersion, "12.")
+	return supportsVirtioFS
+}
+
+func macosNewerThan(targetVersion float64) (bool, error) {
+	osVer, err := syscall.Sysctl("kern.osproductversion")
+	if err != nil {
+		return false, err
+	}
+	logging.Debugf("Mac version is: %s", osVer)
+	osVerArray := strings.SplitAfterN(osVer, ".", 1)
```

Comment:
My understanding of `SplitAfterN( ..., 1)` is that it will return a single value array with the full string.